### DISCUSSION TITLE
The theme Border are choppy and unstable in Obsidian (v. 1.5 .12)

### DISCUSSION THREAD

**Discussion by EvgenA** [Akifyss/obsidian-border#238] [Q&A]
Guys, I want to thank you for creating the Border theme, which I just discovered.
But the fact is that I have a problem and I'm afraid I need help.
On my computer, after installing the theme (Border v.1.7.16), no matter which appearance presets I choose and install, Sunset or Glassmorphism, both are choppy and slow in Obsidian (v. 1.5 .12) on OS Sonoma (v.14.4.1). Obviously, the theme comes with Style Settings, but currently I haven't managed to resolve this really unpleasant phenomenon. Whereas the Blue Topaz theme, for example, or the default Obsidian theme, are fluid and work very well.
Do you have any advice or a solution to remedy this?
I thank you in advance.

**Comment by Akifyss** [comment]
#237
